src/iterative_sorting/iterative_sorting.py

```diff
@@ -31,7 +31,7 @@ def bubble_sort( arr ):
             if arr[j] > arr[j+1]: #checking left and right
                 arr[j], arr[j+1] = arr[j+1], arr[j] #switching
     return arr
-
+something
```

Comment:
This word is causing the iterative unit tests and the Python interpreter to fail.